test/lean/range.expected.lean

```diff
@@ -0,0 +1,27 @@
+import Out.Sail.Sail
+
+open Sail
+
+/-- Type quantifiers: x : Int, 0 ≤ x ∧ x ≤ 31 -/
+def f_int (x : Int) : Int :=
+  0
+
+/-- Type quantifiers: x : Int, 0 ≤ x ∧ x ≤ 31 -/
+def f_nat (x : Int) : Nat :=
+  0
+
+/-- Type quantifiers: x : Int, k_n : Int, 0 ≤ x ∧ x ≤ k_n -/
+def f_negvar (x : Int) : Int :=
+  x
+
+/-- Type quantifiers: x : Int, k_n : Int, 0 ≤ x ∧ x ≤ k_n -/
+def f_nnegvar (x : Int) : Nat :=
```

Comment:
This doesn't go through because it would need an explicit conversion here. I don't immediatly get why the return type is translated as `Nat` but the argument type as `Int` :thinking:

Reply:
See my reply below

Reply:
Which one?

Reply:
[This one](https://github.com/rems-project/sail/pull/869#issuecomment-2598797973)